package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "go-app",
```

Comment:
You should probably change the name of the package to `go-momspeak` (or similar).